src/www/ui_components/storybook.action.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+# Copyright 2018 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eu
+
+web-dev-server --node-resolve --open --watch --root-dir ./src/www
```

Comment:
note that the `.storybook` is in `www` while the `storybook.action.sh` is in `ui_components`. Ideally, everything would live in `ui_components` - however, for us to properly serve the assets we'd need to move them into this folder as well

Ideal for me would be to put the `disc_*.png` files into the new `server_connection_viz` folder so that whole component is self-contained, however, that would require us to update all our build processes to use the `url-loader` or equivalent to support this. follow up pr?

Reply:
In that case I think it makes sense to keep the action with the .storybook file under www/

Reply:
then it clashes with the other start action - should I merge them?